packages/js/components/src/chart/index.js

```diff
@@ -333,6 +333,11 @@ class Chart extends Component {
 		const legendDirection = legendPosition === 'top' ? 'row' : 'column';
 		const chartDirection = legendPosition === 'side' ? 'row' : 'column';
 
+		// Items label is not defined for all the reports.
+		const totalLabel = itemsLabel
```

Comment:
Should we check if it's a string?>

Reply:
Which case would this handle specifically? Just if `itemsLabel` is an empty string, that we still call `sprintf`?

Reply:
I'd say any case other than a non-empty string. [sprintf()](https://github.com/aduth/tannin/blob/630d3e3a00345005d1c06353779ce770ff22719f/packages/sprintf/src/index.js#L107) indeed expects a string.

Reply:
Seems to be some structural issues with this component. Why was `sprintf` being invoked on `itemsLabel` here instead of the function passing in the prop?

Reply:
The use for `sprintf` here is to add the length of the ordered keys into the label, which appears to be only available within this component specifically